### PR TITLE
create database

### PR DESCRIPTION
**Pull request by Muhammed-Ismail2017**
Creating databse and database connector class.
I am waiting for your reviews and suggestions

### CODE REVIEW

**Review by Hegabovic — APPROVED** [review]
Great Work and nice documentation for database part

File: config.php [hunk 1]
```diff
@@ -1 +1,6 @@
 <?php
+define("_driver_", "mysql");
```
Comment (MohamedEldefrawy) [inline config.php:2]: You can use the const keyword in declaring constants instead

**Review by MohamedEldefrawy — APPROVED** [review]
Great work 
Keep it up

File: config.php (hunk 1 above)
Comment (Muhammed-Ismail2017) [reply config.php:2]: I realy appreciate your suggesion

**Pull request merged**